sql/src/main/java/org/apache/druid/sql/calcite/planner/ExplainAttributes.java

```diff
@@ -62,12 +79,48 @@ public String getTargetDataSource()
     return targetDataSource == null ? null : targetDataSource.toString();
   }
 
+  /**
+   * @return the time-based partitioning granularity specified in the <code>PARTITIONED BY</code> clause
+   * for an INSERT or REPLACE statement. Returns null for SELECT/non-DML statements.
```

Comment:
Afaik, SELECT is the only non-DML we use right now, so we can remove the ambiguity.
```suggestion
   * for an INSERT or REPLACE statement. Returns null for SELECT statements.
```